modules/pipeline/pipengine/actionexecutor/plugins/scheduler/executor/plugins/k8sjob/k8sjob.go

```diff
@@ -499,6 +501,47 @@ func (k *K8sJob) generateKubeJob(specObj interface{}) (*batchv1.Job, error) {
 		return nil, errors.Errorf("failed to get cluster info, clusterName: %s, (%v)", k.clusterName, err)
 	}
 
+	if clusterInfo[apistructs.BuildkitEnable] == "true" {
+		delete(clusterInfo, apistructs.BuildkitEnable)
```

Comment:
any concurrent map rw issue?

Reply:
Local variable operations do not require locking.